mscore/inspector/inspector_ambitus.ui

```diff
@@ -878,22 +878,6 @@
  </tabstops>
  <resources/>
  <connections>
-  <connection>
-   <sender>hasLine</sender>
-   <signal>toggled(bool)</signal>
-   <receiver>label_10</receiver>
```

Comment:
this `lable_10`doesn't exist, hence the warning I guess

Reply:
Thanks for explaining all these connection warnings!

Reply:
Needed to understand them myself ;-)